utils/newsletter.py

```diff
@@ -0,0 +1,93 @@
+import calendar
+from datetime import datetime
+from pathlib import Path
+
+import click
+import requests
+
+TEMPLATE = """
+# PythonPune Community Newsletter [{month}-{year} Edition]
```

Comment:
Please move template to a separate file and can we add these blog tag also:
https://raw.githubusercontent.com/pythonpune/meetup-talks/master/content/2020/June/community_news.md

---
title: "PythonPune Community Newsletter [June 2020 Edition]"
date: 2020-06-28T10:51:56+05:30
url: newsletter/june-2020
author: Chandan Kumar
featured_image:
categories:
- Newsletter
tags:
---

Rest looks good